spec/requests/api/rest/admin/gateways_spec.rb

```diff
@@ -3,6 +3,33 @@
 RSpec.describe Api::Rest::Admin::GatewaysController, type: :request do
   include_context :json_api_admin_helpers, type: :gateways
 
+  after { puts pretty_response_json }
```

Comment:
remove this line

it is useful when creating test but should be removed before merge (to not spam into stdout)